src/Build/Collections/Triple.cs

```diff
@@ -18,53 +15,30 @@ namespace Microsoft.Build.Collections
     /// <typeparam name="C">Type of third object</typeparam>
     internal struct Triple<A, B, C>
```

Comment:
This type appears unused. Delete it entirely?

Reply:
Good catch